ansible/roles/secor-telemetry-backup-deploy/defaults/main.yml

```diff
@@ -413,4 +413,23 @@ secor_service_name:
     service_monitor_schedule_hr: "23"
     service_monitor_schedule_min: "57"
     message_channel_identifier: ""
+    message_parser: "com.pinterest.secor.parser.PatternDateMessageParser"
+  cb-audit-telemetry-backup:
```

Comment:
@shobhit-vashistha This specific Secor backup is related to Igot. Should this be merged into the Sunbird Core?